source/dub/dub.d

```diff
@@ -655,10 +703,13 @@ class Dub {
 		}
 
 		if (existing) {
-			if (!ver.startsWith("~") || !(options & FetchOptions.forceBranchUpgrade) || location == PlacementLocation.local) {
+			repoPath.normalize();
+			auto rootNormalized = rootPath;
+			rootNormalized.normalize();
+			if (!ver.startsWith("~") || !(options & FetchOptions.forceBranchUpgrade) || repoPath == rootNormalized) {
```

Comment:
What was the logic behind skipping upgrades for packages that are `PlacementLocation.local`?